README.md

```diff
@@ -213,6 +213,93 @@ oc apply -f deploy/olm-catalog/subscription.yaml
 ```
 
 ### Creating a Windows MachineSet
+
+#### Pre-configuration steps in case of vSphere cluster:
+##### Windows VM assumptions:
+- We tested using updated version of Windows Server 1909 VM image and corresponding container images
+- We assume that the Windows image is configured with SSH so that WMCO can access it. Following are steps to comfigure SSH
+  - Configure SSH on the Windows VM using the [powershell script](https://gist.githubusercontent.com/ravisantoshgudimetla/087bf49088f70edf7f97777386d12f18/raw/0f13048697130a0265ed1050591361fdebcee3ae/powershell.yaml)
+  - Create a new file called C:\Users\Administrator\.ssh\authorized_keys and copy your public key to this file
+- We assume that the VMWare tools has been installed on the Windows VM
+- We assume that the tools.conf at C:\ProgramData\VMware\VMware Tools\tools.conf is having an entry for
+  -`exclude-nics=`
+  
+  This is to ensure that tools won't ignore cloned the vNIC generated on the Windows VM by hybrid-overlay and the vNIC
+  shows in the vCenter console
+- We assume that the Windows image has been sysprep'ed and it is available for cloning:
+`C:\Windows\System32\Sysprep\sysprep.exe /generalize /oobe /shutdown`
```

Comment:
There has to be a `unattend.xml`, this will not work correctly as stated.

Reply:
@jcpowermac  - Updated, ptal